web/help/GuideUnit1.md

```diff
@@ -517,6 +517,16 @@ letters of the function you want, you can narrow down the list.  The type
 signatures tell you what types of information you need to provide to apply
 the function and what type you can expect to end up with.
 
+**Important**: The names of functions are *reserved*. That means you cannot use them as variable names. You would not be able to write program like this:
+
+program=drawingOf(circle)
```

Comment:
Surround your code with five tilde characters to put it in a box with the same coloring and fonts as the editor.  Like this:

```
    ~~~~~
    code here
    ~~~~~
```